pkg/trace/api/normalizer_test.go

```diff
@@ -2,23 +2,29 @@ package api
 
 import (
 	"bytes"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/trace/info"
+	"math/rand"
 	"strings"
 	"testing"
-	"time"
 	"unicode"
 
 	"github.com/DataDog/datadog-agent/pkg/trace/pb"
 	"github.com/stretchr/testify/assert"
 )
 
-func testSpan() *pb.Span {
+func newTagStats() *info.TagStats {
```

Comment:
```
var ts info.TagStats
```
Has the exact same outcome. You don't need this function. Then simply use `&ts` if you need a pointer form.

Reply:
True, updated